docs: update README of the xblock (#18)
eduNEXT/xblock-filesmanager: README.rst

FilesManager XBlock
###################

|status-badge| |license-badge| |ci-badge|

Purpose
*******

Files Manager XBlock is a pluggable extension to the Open edX platform that

> allows course creators to add a file manager to manage files and folders, and

Reviewer: ```suggestion
allows course creators to add a file manager to manage files and folders, and
```
what's manage? can we be more specific?

1. Go to Studio and open the course you want to add the XBlock to.
2. Go to **Settings** > **Advanced Settings** from the top menu.
3. Search for **Advanced Module List** and add ``"filesmanager"`` to the list.
4. Click **Save Changes** button.

   .. image:: https://github.com/eduNEXT/xblock-filesmanager/assets/64033729/5f7c99b8-31d2-492f-8573-88ae8748166b
      :alt: Enable XBlock in a course

Adding a Files Manager Component to a course unit
*************************************************

From Studio, you can add the Files Manager Component to a course unit.

1. Click on the **Advanced** button in **Add New Component**.

   .. image:: https://github.com/eduNEXT/xblock-filesmanager/assets/64033729/7c4cfde4-f2b2-4334-b646-c302dea9c515
      :alt: Open Advanced Components

2. Select **filesmanager** from the list.

   .. image:: https://github.com/eduNEXT/xblock-filesmanager/assets/64033729/d5a524e5-ce0e-4cec-a336-3b04737fb373
      :alt: Select Files Manager Component

Using the Files Manager Component
*********************************

1. Go to edit section of the component from Studio.
2. Create your folders structure by clicking on the **Create folder** button.

> 3. Upload files by clicking on the **Upload files** button.

Reviewer: can we specify that the files uploaded by the xblock are also added to the course assets (so they can be viewed from `Content > Files`)

Author: Done

   .. image:: https://github.com/eduNEXT/xblock-filesmanager/assets/64033729/87b8a552-a8a2-4879-89b0-5fff0d308dab
      :alt: Files Manager Component

   **NOTE:** The soported files types are the same as the ones supported by the
   Open edX platform.

The instructors have an **Unpublished** folder where they can view the files
into the course assets that have not been categorized yet in the component.

> This folder is not visible to the students. In addition, the instructors can

Reviewer: ```suggestion
This folder is not visible to students. In addition, the instructors can
```

Author: Done

preview the files and folders that have been added to the component before
publishing the changes in the course.

View from the Learning Management System (LMS)
**********************************************

> The students can view and download the files and folders that have been added to the

Reviewer: ```suggestion
The students can view and download files and folders that have been added to the
```

Author: Done

component from the LMS. The download can be individual or multiple (as a zip file).

.. image:: https://github.com/eduNEXT/xblock-filesmanager/assets/64033729/fb174233-0b5e-4506-8335-e7197121f2f6
   :alt: View from the LMS

Experimenting with this XBlock in the Workbench
************************************************

`XBlock`_ is the Open edX component architecture for building custom learning
interactive components.

.. _XBlock: https://openedx.org/r/xblock

> You can see the Images Gallery component in action in the XBlock Workbench.

Reviewer: ```suggestion
You can see the Files Manager component in action in the XBlock Workbench.
```

Author: Done

Running the Workbench requires having docker running.

.. code:: bash

    git clone git@github.com:eduNEXT/xblock-filesmanager
    virtualenv venv/
    source venv/bin/activate
    cd xblock-filesmanager
    make upgrade
    make install
    make dev.run

> Once the process is done, you can interact with the Images Gallery XBlock in

Reviewer: ```suggestion
Once the process is done, you can interact with the Files Manager XBlock in
```
You can also fix the rest of the images gallery references

Author: Done
